CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ configure_file(
 # Generates object file (shared library) `libsleef`
 # Defined in src/libm/CMakeLists.txt via command add_library
 set(TARGET_LIBSLEEF "sleef")
+set(TARGET_LIBSLEEFGNUABI "sleefgnuabi")
```

Comment:
I think travis is building it already. It invokes "make install" on all targets.